tests/test_inventory.py

```diff
@@ -0,0 +1,40 @@
+"""Tests for the inventory module."""
+
+from io import BytesIO
+from os.path import join
+from pathlib import Path
+
+import pytest
+
+from mkdocstrings.inventory import Inventory, InventoryItem
+
+try:
+    from sphinx.util.inventory import InventoryFile
+except ImportError:
+    InventoryFile = None  # type: ignore
+
+MKDOCSTRINGS_OBJECTS_INV = Path("site/objects.inv")
+
+
+# https://github.com/bskinn/sphobjinv/blob/260a96cd4eaa7d5cdd8e12f03ae7376243841fa6/conftest.py#L163-L177
+@pytest.mark.skipif(InventoryFile is None, reason="Sphinx is not installed")
```

Comment:
You probably want this `skipif` on all the tests in the module, right?

I usually use an `autouse` fixture to skip a whole test module based on a condition, e.g., [here](https://github.com/bskinn/sphobjinv/blob/master/tests/test_api_good_nonlocal.py#L36-L44).

Something to maybe consider.

Reply:
Indeed, thanks!